### ISSUE TITLE
Request to add a link to Molfeat notebooks for Graphcore IPU

### ISSUE THREAD

**Issue by s-maddrellmander** [datamol-io/molfeat#58]
### 📖 Documentation

I've written two notebooks based on the tutorial that show how to use Molfeat + Graphcore IPUs to fine-tune a pre-trained Transformer model and to train a GINE model. 

Ideally it would be nice to add a short section on the main README to point towards these notebooks. 
Something like this below? I've tried to keep it brief and not like advertising, but please feel free to change as you see fit. 

-------------------------------------------

## Molfeat on the Graphcore IPUs

Two examples showing how to integrate Molfeat with the Graphcore IPUs can be found below. 
* [![Run on Gradient](https://assets.paperspace.io/img/gradient-badge.svg)](https://ipu.dev/lDvDHL)  Training a GNN with Molfeat featurisers 
* [![Run on Gradient](https://assets.paperspace.io/img/gradient-badge.svg)](https://ipu.dev/yoyy6N) Fine-tuning a pre-trained Transformer

-------------------------------------------

Then here for your information are some more links to the notebooks on our Github, on Paperspace where they can be run with access to IPUs, and the Run on gradient link with the shortened URLs. 

The PyTorch Geometric one:
https://github.com/graphcore/Gradient-Pytorch-Geometric/blob/main/molfeat/pytorch_geometric_molfeat.ipynb
https://console.paperspace.com/github/graphcore/Gradient-Pytorch-Geometric?machine=Free-IPU-POD4&container=graphcore/pytorch-geometric-jupyter%3A3.2.0-ubuntu-20.04-20230331&file=molfeat%2Fpytorch_geometric_molfeat.ipynb

[![Run on Gradient](https://assets.paperspace.io/img/gradient-badge.svg)](https://ipu.dev/lDvDHL)

The Transformer one:

https://github.com/graphcore/Gradient-HuggingFace/blob/main/molfeat/transformers_molfeat_finetune.ipynb
https://console.paperspace.com/github/graphcore/Gradient-HuggingFace?machine=Free-IPU-POD4&container=graphcore/pytorch-jupyter%3A3.2.0-ubuntu-20.04-20230331&file=molfeat%2Ftransformers_molfeat_finetune.ipynb
[![Run on Gradient](https://assets.paperspace.io/img/gradient-badge.svg)](https://ipu.dev/yoyy6N)

**Comment by cwognum** [comment]
Hi @s-maddrellmander,

Thank you for showcasing how to use `molfeat` on IPUs! That's exciting! :rocket:

As we love community contributions, we decided to create some dedicate space in our documentation to highlight and celebrate such contributions. This would be the perfect place to then also feature these tutorials! How does that sound? I don't expect this to take us very long and I'll get back to you as soon as we have something to show.

**Comment by maclandrol** [comment]
xref #59

ping @s-maddrellmander
